frontend/src/components/layout/header/WebsitesDropdown.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <DefaultDropdown
+        class="option-select"
+        :items="websites"
+        :value="value"
+        @change="changeWebsite"
+    />
+</template>
+
+<script>
+    import DefaultDropdown from '@/components/common/DefaultDropdown';
+    import {mapActions, mapGetters} from "vuex";
+    import {GET_RELATE_WEBSITES} from "@/store/modules/website/types/getters";
+    import {FETCH_RELATE_WEBSITES} from "@/store/modules/website/types/actions";
+
+    export default {
+        name: "WebsitesDropdown",
+        components: {
+            DefaultDropdown
+        },
+        props: {
+            value: {
+                type:String,
+                required: true
+            },
+        },
+        created() {
+            this.fetchRelateWebsites();
+        },
+        computed: {
+            ...mapGetters('website', {
+                websites: GET_RELATE_WEBSITES
```

Comment:
Don't spread get, fetch and change actions across application. Fetch probably should be done when user authorized. Get and change actions should be done in same component, otherwise it going to be hard to maintain